src/operations/booleans/trees/PolygonTreeNode.js

```diff
@@ -220,6 +215,27 @@ PolygonTreeNode.prototype = {
     }
   },
 
+  clear: function () {
```

Comment:
ugh, sometimes github's view is confusing, where is this one from/used ?

Reply:
It only gets called from Tree.clear()

FYI, I haven't determined if the clear of the tree is necessary. It's not being called at this time.